pkg/monitoring/vmistats/collector_test.go

```diff
@@ -108,12 +108,12 @@ var _ = Describe("Utility functions", func() {
 
 			countMap = makeVMICountMetricMap(nil)
 			Expect(countMap).NotTo(BeNil())
-			Expect(len(countMap)).To(Equal(0))
+			Expect(countMap).To(HaveLen(0))
 
 			vmis := []*k6tv1.VirtualMachineInstance{}
 			countMap = makeVMICountMetricMap(vmis)
 			Expect(countMap).NotTo(BeNil())
-			Expect(len(countMap)).To(Equal(0))
+			Expect(countMap).To(HaveLen(0))
```

Comment:
Same as above for `BeEmpty()` instead of `HaveLen(0)`.

Reply:
Done. Thanks.